scaling_performance/host_practices.adoc

```diff
@@ -57,42 +57,94 @@ include writing a large amount of log data.
 etcd is a distributed key-value store that {product-title} uses for
 configuration.
 
+Version 3 of etcd brings with it important scalability and performance improvements that reduce CPU, memory, network and disk requirements for any size cluster.  OpenShift 3.5 introduced etcd version 3.1.
```

Comment:
`Version 3 of etcd introduces a new storage version which brings with it important scalability and performance improvements that reduce CPU, memory, network and disk requirements for any size cluster.`

Trying to differentiate between package version and storage api.

I'd get rid of the last sentence, while they may have been upgraded to etcd 3.1, we don't use v3 storage there so i'd just skip it.